lib/libc/mingw/math/arm/sincos.S

```diff
@@ -8,11 +8,8 @@
 	.file	"sincos.S"
 	.text
 	.align 2
-	.globl __MINGW_USYMBOL(sincos)
```

Comment:
There's a high likelihood of this change being lost in a future MinGW-w64 source import. To prevent that, add a line like this:

```asm
/* zig patch: remove sincos symbol */
```